apps/site/src/app/admin/participants/components/ParticipantsTable.tsx

```diff
@@ -101,10 +198,73 @@ function ParticipantsTable({
 			trackBy="_id"
 			empty={emptyMessage}
 			filter={
-				<TextFilter filteringPlaceholder="Find participants" filteringText="" />
+				<>
+					<TextFilter
```

Comment:
Chore: These filters currently take up three separate lines, is there a way to make them all show up in one line like the Cloudscape's [Table with select filter](https://cloudscape.design/examples/react/table-select-filter.html) demo?
Could we also extract these filters to a separate component file as the current file is getting a bit lengthy?